src/components/CodeEditor/yaml-lint.js

```diff
@@ -0,0 +1,26 @@
+import CodeMirror from 'codemirror';
+import { safeLoad } from 'js-yaml';
+import 'codemirror/addon/lint/lint';
+
+/*
+  Override 'codemirror/addon/lint/yaml-lint' registerHelper
+  so that it doesn't use window.jsyaml
+  https://github.com/codemirror/CodeMirror/blob/master/addon/lint/yaml-lint.js
+ */
+CodeMirror.registerHelper('lint', 'yaml', text => {
```

Comment:
This could be rewritten as:

```js
CodeMirror.registerHelper('lint', 'yaml', text => {
  try {
    safeLoad(text);
    return [];
  } catch (e) {
    const loc = e.mark;
 
    return [{
      from: CodeMirror.Pos(loc.line, loc.column),
      to: CodeMirror.Pos(loc.line, loc.column),
      message: e.message,
    }];
  }
});
```